### PR TITLE
Show identifications in viewer 177576899

### PR DESCRIPTION
**Pull request by MLumme**

### CODE REVIEW

File: projects/laji/src/app/shared-modules/document-viewer/identifications/identification/identification.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        [tooltip]="'showSpeciesCard' | translate"
+        [placement]="'right'"
+        container="body"
+        target="_blank" [routerLink]="['/taxon', ((identification.linkings.taxon.id || identification.linkings.taxon.qname) | toQName)] | localize" (click)="onTaxonLinkClick($event)">
+        <i class="glyphicon glyphicon-tag"></i>
+      </a>
+    </span>
+  </span>
+  <ng-container *ngIf="showFacts">
+    <div class="inset">
```
Comment (aorin) [inline projects/laji/src/app/shared-modules/document-viewer/identifications/identification/identification.component.html:18]: Laji ui project has some helper classes such as ml-5 (gives 18px margin left) that could be handy here

**Review by aorin — CHANGES_REQUESTED** [review]
Dev branch should be merged to this as it seems to have some conflicts. Otherwise looks good :thumbsup:. I added some small comments.

**Comment by MLumme** [comment]
@aorin Finally implemented those changes after some docker problems, didn't see anything break, but might want to check also to be sure.

**Review by aorin — APPROVED** [review]
:+1:

**Pull request merged**